### DISCUSSION TITLE
camu.select trouble

### DISCUSSION THREAD

**Discussion by christophertrapani** [Ircam-RnD/catart-mubu#89] [Bug, Issues, and Suggestions]
<img width="813" alt="Screenshot 2023-05-10 at 10 42 59 PM" src="https://github.com/ircam-ismm/catart-mubu/assets/35881355/796e9d74-8384-46af-bb74-c5fc3f999dd6">

Screenshot from tutorial 3: Combined Selection with default "Gouts" file loaded (chop segmentationP

column 1 (frequency mean) sent to camu.select (verified in weights), followed by select 450. camu.select returns marker index 44 — but marker 44 value in column 1 is nowhere near 450., there are several other closer choices...

something off with selection?

**Comment by diemoschwarz** [comment]
In that tutorial, freq is column 0 (no duration)

**Comment by christophertrapani** [comment]
Thanks. Missed that after 12 hours of staring at CaMu yesterday...
